src/test/java/com/palantir/gradle/circlestyle/CircleStylePluginTests.java

```diff
@@ -35,8 +25,17 @@
 import org.junit.contrib.java.lang.system.EnvironmentVariables;
 import org.junit.rules.TemporaryFolder;
 
-import com.google.common.base.Joiner;
-import com.google.common.io.Files;
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.google.common.base.Charsets.UTF_8;
```

Comment:
move static import back to the top of the file

Reply:
hmm my IDE hates this and keeps moving it back down here